src/sass/fonts.scss

```diff
@@ -0,0 +1,20 @@
+@font-face {
+  font-family: 'Mulish';
+  font-style: normal;
+  font-weight: normal;
+  src: url(https://dist.myworkout.com/fonts/Mulish-Regular.ttf);
+}
+
+@font-face {
+  font-family: 'Mulish';
+  font-style: normal;
+  font-weight: bold;
+  src: url(https://dist.myworkout.com/fonts/Mulish-Bold.ttf);
+}
+
+@font-face {
+  font-family: 'Baloo 2';
+  font-style: normal;
+  font-weight: bold;
+  src: url(https://dist.myworkout.com/fonts/Baloo2-Bold.ttf);
+}
```

Comment:
This can be out of the styles library maybe. It's only being used in the webapp 🤔

Reply:
Fonts that are not used won't be imported, so this is fine 👍🏻